sdk/go/x/auto/local_workspace.go

```diff
@@ -520,7 +528,7 @@ type GitRepo struct {
 
 // SetupFn is a function to execute after enlisting in a git repo.
 // It is called with a PATH containing the pulumi program post-enlistment.
```

Comment:
Can we update the docstring to indicate that this receives the full workspace after all other options have been processed (rather than the path).